app/payment-request-mapping/payment-request-schema.js

```diff
@@ -0,0 +1,23 @@
+const Joi = require('joi')
+
+module.exports = Joi.object({
+  sourceSystem: Joi.string().optional(),
+  deliveryBody: Joi.string().optional(),
+  invoiceNumber: Joi.string().optional(),
```

Comment:
I don't think `optional()` works with strings.  I think you have to use `.allow('')`